content/theme/characters/archetypes/jedi/jedi-guardian.md

```diff
@@ -2,5 +2,38 @@
 title: "Jedi Guardian"
 date: 2020-11-09T21:18:13-08:00
 draft: false
----
+layout: archetype
+ attributes:
+   awareness: 8
+   coordination: 8
+   knowledge: 7
+   physique: 11
+   presence:7
+   reflex: 11
+
+ skills:
+   awareness:
+     search: 2
+   coordination:
+     pilot: 1
+   knowledge:
+     investigate: 2
+     medicine: 2 
+   physique:
+     strength: 3
+     toughness: 4
+     stamina: 4
+   presence:
+     willpower: 3
+     intimidate: 2
+     lead: 3
+   reflex:
+     acrobatics: 4
+     dodge: 3
+     leap: 3
+     melee: 4
+ ---
```

Comment:
Sorry, also should not be a leading space before the front-matter delimiter `---`
Had to do some searching to find out why the build was failing.